ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/intended/structured_configs/DC1-SVC3A.yml

```diff
@@ -914,6 +914,13 @@ port_channel_interfaces:
     mlag: 19
     lacp_fallback_mode: static
     lacp_fallback_timeout: 10
+  Port-Channel22:
+    description: CUSTOM_server15_port_channel_disabled_interfaces_
```

Comment:
It seems you edited these files. These should be generated through molecule. Let me know if you need any help.

Reply:
These files were edited through molecule!

Reply:
I changed the inventory in the molecule scenario, ran the molecule test and commited the changed files.

Reply:
Ok. I haven't seen ```_``` at the end of description. So I thought something was wrong. You are good, if molecule is used.